blog/20140130-mate-desktop-live-cd.de.md

```diff
@@ -26,27 +26,27 @@ Wenn Sie die zusätzlichen Bytes verschmerzen können, lassen Sie bitte den Clie
 nachdem der Download beendet ist, damit Sie es an andere zurück "seeden" können.
 *Für höchste Download-Geschwindigkeit wird ein "web-seed-fähiger" Client empfohlen.
 
-  * [matelivecd-14.069.1449.iso.torrent](http://repo.mate-desktop.org/livecd/matelivecd-14.069.1449.iso.torrent)
+  * [matelivecd-14.069.1449.iso.torrent]
 
 ### Direkter HTTP-Download
 
 Zusätzlich zu dem BitTorrent-Link oben, kann das ".iso" der Live-CD auch
 über HTTP heruntergeladen werden. Bitte vergewissern Sie sich, dass die heruntergeladene
 Datei zu der SHA1-Checksum passt.
 
-  * [matelivecd-14.069.1449.iso](http://repo.mate-desktop.org/livecd/matelivecd-14.069.1449.iso) [ ~964MB ]
+  * [matelivecd-14.069.1449.iso]
     * SHA1: `12b529d17d1ecef39d9b0e1122fb5271e0534c0a`
 
 ## Kompatibilität
 
-Die Live-CD wurde mit [Arch Linux](http://www.archlinux.org) und MATE 1.8 gebaut. 
+Die Live-CD wurde mit [Arch Linux](https://www.archlinux.org) und MATE 1.8 gebaut. 
 Die Live-CD ist 32-Bit und sollte daher auf jedem i686- oder x86_64-Rechner mit
 mindestens 512MB RAM laufen. X.Org-Treiber sind enthalten für Intel (`i915`),
 AMD/ATI (`radeon`) und nvidia (`nouveau`) mit einer Ausweichlösung mittels VESA.
 Treiber für Virtual Box und VMware sind ebenso enthalten, damit das Ausprobieren
 auf diesen Virtualisierungs-Lösungen einfach ist.
 
-Alle gängigen Dateisysteme, inclusive [ZFS](http://open-zfs.org), werden 
+Alle gängigen Dateisysteme, inclusive [ZFS], werden 
```

Comment:
The link to OpenZFS home page was removed.

Reply:
Link is back